kalite/distributed/backup.py

```diff
@@ -0,0 +1,34 @@
+import dbbackup
+import datetime
+import os
+import sys
+import time
+
+from django.core.management import call_command
+from django.conf import settings
+
+from fle_utils.chronograph.models import Job
+from fle_utils.chronograph.utils import force_job
+
+DAYS = 7
+SECONDS = 86400
+
+def file_rename():
+    defaultfilename ='default.backup'
+    time = datetime.datetime.now().ctime()
+    time = time + '.backup'
+    defaultpath = os.path.join(settings.BACKUP_DIRPATH, defaultfilename)
+    target = os.path.join(settings.BACKUP_DIRPATH, time)
+    if(os.path.exists(defaultpath)):
+        os.rename(defaultpath, target)   
+
+def setup_backup():
+    command = call_command('kalitebackup')
```

Comment:
I don't think this will work :) Python does eager evaluation, meaning `call_command('kalitebackup')` already starts executing by this line. By the time you get to the `force_job` line, the backup already happened.

Instead, what `force_job` is expecting is simply the name of the command, and then it'll use `call_command` automatically when it's time to run it. Call it like this:

`force_job("kalitebackup", frequency="DAILY")`